tests/conftest.py

```diff
@@ -1544,6 +1544,22 @@ def collect_db_dump(request, duthosts):
     yield
     collect_db_dump_on_duts(request, duthosts)
 
+@pytest.fixture(autouse=True)
+def verify_new_core_dumps(duthost):
+    if "20191130" in duthost.os_version:
+        pre_existing_cores = duthost.shell('ls /var/core/ | grep -v python | wc -l')['stdout']
+    else:
+        pre_existing_cores = duthost.shell('ls /var/core/ | wc -l')['stdout']
+    
+    yield
+    if "20191130" in duthost.os_version:
+        coredumps_count = duthost.shell('ls /var/core/ | grep -v python | wc -l')['stdout']
+    else:
+        coredumps_count = duthost.shell('ls /var/core/ | wc -l')['stdout']
+    if int(coredumps_count) > int(pre_existing_cores):
+        pytest.fail("Core dumps found. Expected: {} Found: {}. Test failed".format(pre_existing_cores,\
+            coredumps_count))
```

Comment:
Is it better to print new file name here?  If it fails, we are able to know which one is new core dump file and the timestamp could tell us when DUT generates file.

Reply:
I think it's unnecessary to print new file name. We won't collect core dumps to docker, if we want to know details of core dumps, we need to ssh to DUT, so I think it's unnecessary to find new file and print its name if we can simply use "ls -l" to get it.